app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -449,6 +458,27 @@ class WireNotificationManager @Inject constructor(
         }
     }
 
+    private fun verifyIfShouldPlayPingSound(
```

Comment:
`verify` would be a better name if it returned a `boolean` and didn't have any side effects :)

```suggestion
    private fun playPingSoundIfNeeded(
```